server/services/interventionsService.test.ts

```diff
@@ -939,6 +1009,56 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
 
+  describe('setComplexityLevelForServiceCategory', () => {
+    it('returns the updated referral when selecting a complexity level on a cohort referral', async () => {
+      await provider.addInteraction({
+        state: `There is an existing draft cohort referral with ID of 06716f8e-f507-42d4-bdcc-44c90e18dbd7, and it has had multiple service categories selected`,
+        uponReceiving: 'a PATCH request to set the complexity level for a service category on a referral',
+        withRequest: {
+          method: 'PATCH',
+          path: `/draft-referral/06716f8e-f507-42d4-bdcc-44c90e18dbd7/complexity-levels`,
```

Comment:
should the trailing `s` be here on this endpoint?

Reply:
you can only set one at a time right?

Reply:
I went back and forth on this, mostly in terms of RESTful naming I thought plurals was the way to go, I know we had a bunch of conversations about this back in the day! - I don't think we've been all that consistent, although we do mostly refer to a single `draft-referral` here, so I'm happy to switch to singular here if you think it's more logical?

Reply:
i don't really mind. the only reason i commented is because for the outcomes one you are passing 1 or more outcome per service cat. for this one you are just passing one everytime.

Reply:
You're right - I'll switch it, no biggie!